docs/EN_US/ContainerizedHPCC/ContainerizedMods/TerraformDeployChap.xml

```diff
@@ -0,0 +1,614 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE section PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN"
+"http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd">
+<chapter id="HPCC_Terraform_deployment">
+  <title>Deploying HPCC Systems<superscript>®</superscript> with
+  Terraform</title>
+
+  <para>Manual deployments can be error-prone and inconsistent. As your
+  deployments become more customized and your need for additional resources
+  grows it can become exponentially more difficult and time consuming.</para>
+
+  <para>Fortunately, there are multiple IaC (infrastructure as code)
+  orchestration tools available that can simplify the deployment process. One
+  of those orchestration tools is Terraform. This chapter provides
+  instructions on using Terraform modules to deploy an HPCC Systems instance
+  specifically on the Azure Cloud.</para>
+
+  <para>These modules were developed by the HPCC Systems platform team for
+  general open-source community usage. You may require specific customizations
+  for your particular needs. For example, your organization may require
+  opinionated modules for production systems. You can develop your own
+  customized modules, per your requirements and utilize them in the same
+  manner outlined here.</para>
+
+  <sect1 id="GS_DeployingHPCC_w_Terraform" role="nobrk">
+    <title>Getting Started With Terraform </title>
```

Comment:
Should we change the title to something like "HPCC-Platform Interactive Terraform Deployment"?

Reply:
went with "Interactive Terraform Deployment" that makes much more sense. Thanks.